tests/php/extra/Footnotes.text

```diff
@@ -50,7 +50,7 @@ footnote test[^reference].
 	This footnote has a footnote of its own.[^nested]
 
 [^nested]:
-	This footnote should appear even though as it is refered
+	This footnote should appear even though as it is referred
```

Comment:
The `php` test files are borrowed directly from the PHP project and should always remain unchanged. Please do not make any changes to these files. These changes need to be undone before I can accept this.